models/univar_lstm_model/univar_lstm_model_run.py

```diff
@@ -51,6 +53,13 @@ def main(config: DictConfig) -> None:
         batch_size=1
     )
 
+    # Get predictions
+    path_to_results = SDKConfig().get_output_dir("plots_model_results")
```

Comment:
"plots_univar_lstm_model_results" would be a better name, becasue TJ is plotting arima results as well. Or you can select a better name, that includes "univar", then we use the same for "multivar".